src/nifax/control/util/ColumnSorter.java

```diff
@@ -0,0 +1,46 @@
+package nifax.control.util;
+
+import java.util.Comparator;
+import java.util.Vector;
+
+/**
+ *
+ * @author NB
+ */
+public class ColumnSorter implements Comparator {
+
+    int colIndex;
+
+    public ColumnSorter(int colIndex) {
+        this.colIndex = colIndex;
+    }
+
+    @Override
+    public int compare(Object a, Object b) {
```

Comment:
I'm not sure if compare should return an integer, normally if you compare something you want to know only the result of the comparation